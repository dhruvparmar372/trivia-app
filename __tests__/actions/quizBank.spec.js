import {
  addQuiz as addQuizAction,
  fillQuizBank as fillQuizBankAction
} from "source/actions/quizBank";

const dummyQuiz = {
  id: 1,
  questions: [
    {
      id: 1,
      type: "boolean",
      question: "1 + 1 = 2?",
      correct_answer: "True",
      incorrect_answers: ["False"]
    }
  ]
};

describe("quizBank actions", () => {
  test("add quiz action", () => {
    expect(addQuizAction(dummyQuiz)).toMatchSnapshot();
  });

  test("fill quiz bank action", () => {
    expect(fillQuizBankAction(2)).toMatchSnapshot();
  });
});
