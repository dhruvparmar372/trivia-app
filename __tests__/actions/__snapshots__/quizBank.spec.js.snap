// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`quizBank actions add quiz action 1`] = `
Object {
  "quiz": Object {
    "id": 1,
    "questions": Array [
      Object {
        "correct_answer": "True",
        "id": 1,
        "incorrect_answers": Array [
          "False",
        ],
        "question": "1 + 1 = 2?",
        "type": "boolean",
      },
    ],
  },
  "type": "QUIZBANK/ADD",
}
`;

exports[`quizBank actions fill quiz bank action 1`] = `
Object {
  "quizCount": 2,
  "type": "QUIZBANK/FILL",
}
`;
